### ISSUE TITLE
GAME_TARGET message can cause peer client error on reconnect

### ISSUE THREAD

**Issue by Susucre** [magefree/mage#11155]
I have no idea what it means or how to fix, but can be related to the double trigger error from #11052

![image](https://github.com/magefree/mage/assets/34709007/107b8c60-3723-4dc4-93b1-291969ca8f3f)

How to reproduce (not sure it is 100%, did it twice in a row):
-> Have a two player game, player A cast Kroxa
-> With the pick triggered ability open, start a new client with player A name, to reconnect in the game.

```java
java.lang.NullPointerException: peer
	at sun.awt.windows.WInputMethod.openCandidateWindow(Native Method)
	at sun.awt.windows.WInputMethod.access$400(WInputMethod.java:45)
	at sun.awt.windows.WInputMethod$1.run(WInputMethod.java:602)
	at java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:311)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at mage.client.dialog.MageDialog.startModal(MageDialog.java:173)
	at mage.client.dialog.MageDialog.show(MageDialog.java:83)
	at java.awt.Component.show(Component.java:1671)
	at java.awt.Component.setVisible(Component.java:1623)
	at javax.swing.JComponent.setVisible(JComponent.java:2644)
	at mage.client.dialog.MageDialog.setVisible(MageDialog.java:89)
	at mage.client.game.FeedbackPanel.updateConnectedDialog(FeedbackPanel.java:203)
	at mage.client.game.FeedbackPanel.updateOptions(FeedbackPanel.java:189)
	at mage.client.game.FeedbackPanel.prepareFeedback(FeedbackPanel.java:121)
	at mage.client.game.GamePanel.pickTarget(GamePanel.java:1657)
	at mage.client.remote.CallbackClientImpl.lambda$processCallback$0(CallbackClientImpl.java:220)
	at java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:311)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by Susucre** [comment]
Step after that, ordering the trigger, still got no battlefield or players panel:
![image](https://github.com/magefree/mage/assets/34709007/bc30675b-51e4-41f9-b731-94c421c1558c)

**Comment by JayDi85** [comment]
fixed by #11189
